### DISCUSSION TITLE
how do we store cooltools' outputs? One idea: inside cooler files

### DISCUSSION THREAD

**Discussion by golobor** [open2c/cooltools#284] [Ideas]

**Comment by golobor** [comment]
ping @nvictus - this would unblock the development of eigdecomp

**Comment by mimakaev** [comment]
I think this is not moving forward because we don't have a good definition of expected (by chr, genome-wide etc). Also, we're not sure about by-chromosome vs by-arm: what is better, and where to get centromere positions.

**Comment by Phlya** [comment]
Another related issue, https://github.com/mirnylab/cooler/issues/122

**Comment by golobor** [comment]
we arrived to a solution, where external tools (including cooltools) are allowed to store their information as tables in groups /ext/<tool_name>.
@nvictus , do we need to take any further action to close this item? Maybe, a single line in the cooler schema?

**Comment by gfudenberg** [comment]
what about a cli command for cleaning the cooler's extra tables? "cooler
purge tables" or "cooler refresh" , with options to keep specific keys
(e.g. purge expected, but not weights from balancing)
